product_pricelist_brand_required/README.rst

```diff
@@ -0,0 +1,69 @@
+================================
+Product Pricelist Brand Required
+================================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:013bd3a58c39d2154ea63fa1eb72fca51faebc2a5538cbdf673e0afd8f8f3c47
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-sygel--technology%2Fsy--product--attribute-lightgray.png?logo=github
+    :target: https://github.com/sygel-technology/sy-product-attribute/tree/17.0/product_pricelist_brand_required
+    :alt: sygel-technology/sy-product-attribute
+
+|badge1| |badge2| |badge3|
+
+Check if the product's brand matches any selected brand in the pricelist
+item.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+1. Create a Pricelist Item.
+2. Add the product brands you want to include in the pricelist item.
```

Comment:
Could you explain what how does it work when a pricelist item has brands? Thank you (and sorry for not telling before).